includes/ting_search_autocomplete.admin.inc

```diff
@@ -0,0 +1,185 @@
+    '#default_value' => $default_settings['sort'],
+  );
+
+  $form['ting_search_autocomplete_settings']['agency'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Agency', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['agency'],
+  );
+
+  $form['ting_search_autocomplete_settings']['profile'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Profile', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['profile'],
+  );
+  $form['ting_search_autocomplete_settings']['maxSuggestions'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Max suggestions', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['maxSuggestions'],
+  );
+
+  $form['ting_search_autocomplete_settings']['maxTime'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Max request time', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['maxTime'],
+  );
+
+  $form['ting_search_autocomplete_settings']['highlight'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Highlight', array(), array('context' => 'ting_search_autocomplete')),
+    '#description' => t('If checked, search prefix will be enclosed in highlight prefix and suffix', array(), array('context' => 'ting_search_autocomplete')),
+    '#default_value' => $default_settings['highlight'],
+    '#return_value' => 'true',
+  );
+
+  $form['ting_search_autocomplete_settings']['highlight.pre'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Highlight prefix', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['highlight.pre'],
+  );
+
+  $form['ting_search_autocomplete_settings']['highlight.post'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Highlight suffix', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['highlight.post'],
+  );
+
+  $form['ting_search_autocomplete_settings']['minimumString'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Minimum string', array(), array('context' => 'ting_search_autocomplete')),
+    '#description' => t('Minimum string length to do autocomplete lookup on.', array(), array('context' => 'ting_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['minimumString'],
+  );
+
+  $form['ting_search_autocomplete_help'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Autocomplete help', array(), array('context' => 'ting_search_autocomplete')),
+    '#tree' => TRUE,
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );
+
+  $form['ting_search_autocomplete_help']['description'] = array(
+    '#theme' => 'table',
```

Comment:
Could you remind me why this is implemented as a table and not as `#description` for the individual form elements?